tests/flow/test_function_calls.py

```diff
@@ -395,3 +395,28 @@ def test18_allocated_keys(self):
         expected_result = [['STR1', [1, 2, 3]],
                            ['STR2', [1, 2, 3]]]
         self.env.assertEquals(actual_result.result_set, expected_result)
+
+    def test19_has_labels(self):
```

Comment:
Add a test which calls `hasLabels` directly, e.g. `RETURN hasLabels(n, ['A', 'B'])`
test for invalid inputs e.g. `RETURN hasLabels(n, ['A', 'B', 2])`